andela/Ah-backend-groot
# #164069308  User email verification

**What does this PR do?**

* Cleans up the  functionality that sends an email to a user after registering with the Authors Heaven.

**Description of the tasks to be completed?**
* Sends verification email with activation link to user for activating his created account on Authors Heaven.

**How should this be manually tested?**
* Install project requirements using `pip install -r requirements.txt.
* Sign up with the Authors Heaven at /api/users/  
sample data =  {
"user" : {
    "username": "example23",
    "email": "example@gmail.com,   
    "password": "A123456678av$"
}
}
* Check provided email address for email verification link.

**What are the relevant pivot tracker stores?**
* #164069308
